tests/test_compute.py

```diff
@@ -31,6 +31,18 @@
 SERVICE_ENDPOINT = BaseURLs.BASE_PROVIDER_URL + '/services/download'
 
 
+def test_compute_expose_endpoints(client):
+    get_response = client.get('/')
+    result = get_response.get_json()
+    assert 'servicesEndpoints' in result
+    assert 'software' in result
+    assert 'version' in result
+    assert 'network-url' in result
+    assert 'provider-address' in result
+    assert get_response.status == '200 OK'
+    assert len(result['servicesEndpoints']) == 4
```

Comment:
maybe check for each service that we know we have in this version of provider..  it's longer, but safer

Reply:
Ok, I will do that. Thank you for feedback.